js/main.js

```diff
@@ -153,3 +157,36 @@ $ul.addEventListener('click', function (event) {
     }
   }
 });
+
+const $popup = document.querySelector('.popup-container');
+const $grayBackground = document.querySelector('.gray-background');
+
+$delete.addEventListener('click', function (event) {
+  event.preventDefault();
```

Comment:
Not needed. You only need this on `submit` events to prevent the page from reloading